src/mcts/search.cc

```diff
@@ -756,8 +756,12 @@ void SearchWorker::ExecuteOneIteration() {
   if (params_.GetMaxConcurrentSearchers() != 0) {
     while (true) {
       // If search is stop, we've not gathered or done anything and we don't
-      // want to, so we can safely skip all below.
-      if (search_->stop_.load(std::memory_order_acquire)) return;
+      // want to, so we can safely skip all below. But make sure we have done
+      // at least one iteration.
+      if (search_->stop_.load(std::memory_order_acquire) &&
+          search_->total_playouts_ > 0) {
```

Comment:
`search_->total_playouts_ + search->initial_visits_`, and it will also fix #996

Reply:
search->GetTotalPlayouts() + search->initial_visits_
to avoid the mutex protection warning issue.